integrationTests/relayers/ethToMultiversXWithChainSimulator_test.go

```diff
@@ -193,15 +206,26 @@ func TestRelayersShouldExecuteTransfersFromEthToMultiversXWithChainSimulator(t *
 		pk: ownerPK,
 		sk: ownerSK,
 	}
+	erc20ContractsHolder := createMockErc20ContractsHolder([]common.Address{ethGenericTokenAddress}, ethSafeAddress, []*big.Int{mintAmount})
```

Comment:
we can use a real component of Erc20SafeContractsHolder

Reply:
pushed